src/main/java/com/iota/iri/IRI.java

```diff
@@ -220,6 +222,10 @@ private static void validateParams(final Configuration configuration, final Stri
         if (vsendLimit != null) {
             configuration.put(DefaultConfSettings.SEND_LIMIT, vsendLimit);
         }
+        if (parser.getOptionValue(sync) != null) {
+            configuration.put(DefaultConfSettings.NEW_TX_LIMIT, "0.0");
```

Comment:
This should be "9.0" according to the default value further below?

Reply:
this is intended for syncing nodes.
they can start their node w/ `--sync` to bypass the `new_tx_limit`.
(this isn't needed on normal "older" nodes, b/c the syncing node wont send a lot of *new* transactions while syncing)